dashboard/test/models/script_test.rb

```diff
@@ -287,4 +287,24 @@ def setup
 
     assert_equal 1, script.summarize[:stages].count
   end
+
+  test 'should generate PLC objects' do
+    @script_file = File.join(self.class.fixture_path, 'test_plc.script')
+    scripts, custom_i18n = Script.setup([@script_file])
```

Comment:
Nit: this doesn't need to be an `@instance` variable but this follows the existing pattern in this file.